server/util/twilio.js

```diff
@@ -2,16 +2,32 @@ import Twilio from 'twilio';
 
 // Account SID and Auth Token are stored as environmental variables
 // Twilio Node module will check for them automatically upon initialization
-//
-//
-//
-// UNCOMMENT THE BELOW TWO LINES WHEN MERGING THE PLACE CALLS PR
-// |
-// |
-// |
-// V
-// const twilioClient = new Twilio();
-// const CALLER_ID = process.env.TWILIO_CALLER_ID;
+const twilioClient = new Twilio();
+
+const CALLER_ID = process.env.TWILIO_CALLER_ID;
+const DEV_PATH = process.env.DEV_PATH;
+
+export function callVolunteer(user_id, campaign_id, phone_number) {
+  return new Promise((resolve, reject) => {
+    twilioClient.calls.create({
+      url: `${DEV_PATH}/users/${user_id}/campaigns/${campaign_id}/calls/start`,
+      to: phone_number,
+      from: CALLER_ID,
+      statusCallback: `${DEV_PATH}/users/${user_id}/callback`,
+      statusCallbackMethod: 'POST',
+      statusCallbackEvent: ['completed', 'busy', 'no-answer', 'canceled', 'failed'],
```

Comment:
Looking at the API spec, seems that the possible events are initiated, ringing, answered, and completed. And completed is the default (which captures all possible ways to complete the call). So we can omit this param.

Reply:
Got it. Thanks!